README.md

```
@@ -9,7 +9,51 @@
+Models created in Blender
+
+Voice editing via Audacity
+
+Motion and combat animations via Mixamo
+
+Version control using GitHub + Git LFS
+
+UI crafted with TextMeshPro and Unity UI Toolkit
+
+Game Link:
+https://notorious-dhaka.itch.io/kujola
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)